src/main/resources/application.yml

```diff
@@ -17,6 +17,15 @@ spring:
     serialization:
       WRITE_DATES_AS_TIMESTAMPS: false
 
+  security:
```

Comment:
I guess this is unrelated to the Spring security move but this is what you started of doing?

Reply:
Yes - started off getting the signing key from the jwk-set-uri location (well, Spring does that for you).  But then added issuer and moved to Spring Security 5 after feedback.